jetty-security/src/main/java/org/eclipse/jetty/security/authentication/SessionAuthentication.java

```diff
@@ -92,23 +90,18 @@ private void readObject(ObjectInputStream stream)
     }
 
     @Override
+    @Deprecated
     public void logout()
     {
-        if (_session!=null && _session.getAttribute(__J_AUTHENTICATED)!=null)
-            _session.removeAttribute(__J_AUTHENTICATED);
-
-        doLogout();
     }
 
+
+    @Deprecated
```

Comment:
don't need to deprecate a private method. Just remove

Reply:
Done.